### PR TITLE
Fungible tokens update, UI updates, Seed bug remove, Translations updates

### PR DESCRIPTION
**Pull request by vmangelovv**
- Fix bug in reveal seed phrase. #230 

- Update Translations for all langs #235 & #222 

- Import the new package (aeternity-tokens) for implemented extensions for deployed contracts and UI to it.

- Fix bug in amount displaying #232

### CODE REVIEW

File: src/popup/router/pages/AccountPassword.vue [hunk 1]
```diff
@@ -34,6 +34,8 @@ import { MINPASSWORDLENGTH } from '../../utils/constants';
 import Password from 'vue-password-strength-meter';
 import { mapGetters } from 'vuex'
 import { redirectAfterLogin } from '../../utils/helper';
+const Cryptr = require('cryptr');
+const cryptr = new Cryptr('myTotalySecretKey');
```
Comment (mradkov) [inline src/popup/router/pages/AccountPassword.vue]: @vmangelovv 

Is this key needed for decryption?
- If not we can use random buffer for seed here.
- Else why don't we use the user's password here?

**Comment by mradkov** [comment]
These issues #243 && #244 are related to this PR. Suggest we fix those before approval and merging.

File: src/popup/router/pages/Send.vue [hunk 2]
```diff
@@ -242,7 +242,7 @@ export default {
           let tx = {
             popup:false,
             tx: {
-              amount:this.form.amount,
+              amount:amount,
```
Comment (petbaik) [inline src/popup/router/pages/Send.vue]: This change leads to wrong display of entered amount in Send Page
image
image
Amount in second picture should be **1 AE**

**Comment by petbaik** [comment]
> These issues #243 && #244 are related to this PR. Suggest we fix those before approval and merging.

fix #243 in https://github.com/waellet/waellet/pull/254/commits/7b469fe058c1112dcbd4da902cff655ec142483c

**Comment by mradkov** [comment]
> > These issues #243 && #244 are related to this PR. Suggest we fix those before approval and merging.
> 
> fix #243 in [7b469fe](https://github.com/waellet/waellet/commit/7b469fe058c1112dcbd4da902cff655ec142483c)

Ready to merge?

**Comment by petbaik** [comment]
> > > These issues #243 && #244 are related to this PR. Suggest we fix those before approval and merging.
> > 
> > 
> > fix #243 in [7b469fe](https://github.com/waellet/waellet/commit/7b469fe058c1112dcbd4da902cff655ec142483c)
> 
> Ready to merge?

Just need to push several commits before merge

File: src/popup/router/pages/AccountPassword.vue (hunk 1 above)
Comment (petbaik) [reply src/popup/router/pages/AccountPassword.vue]: This is fixed in https://github.com/vmangelovv/aepp-waellet/commit/972a6c713f8da8a665eac47952e8c6f660ca7768. Now we are using webcrypto and user's password for encryption and decryption of the seed

File: src/popup/router/pages/Send.vue (hunk 2 above)
Comment (petbaik) [reply src/popup/router/pages/Send.vue]: Fixed in https://github.com/vmangelovv/aepp-waellet/commit/972a6c713f8da8a665eac47952e8c6f660ca7768

**Pull request merged**